installers/metrics/ansible/roles/pgo-metrics/files/prometheus.yml

```diff
@@ -0,0 +1,63 @@
+---
+global:
+  scrape_interval: 30s
+  scrape_timeout: 30s
+  evaluation_interval: 5s
+
+scrape_configs:
+- job_name: crunchy-collect
+  kubernetes_sd_configs:
+  - role: pod
+
+  relabel_configs:
+  - source_labels: [__meta_kubernetes_pod_label_crunchy_collect]
+    action: keep
+    regex: true
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 5432
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 10000
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 8009
+  - source_labels: [__meta_kubernetes_pod_container_port_number]
+    action: drop
+    regex: 2022
+  - source_labels: [__meta_kubernetes_namespace]
+    action: replace
+    target_label: kubernetes_namespace
+  - source_labels: [__meta_kubernetes_pod_name]
+    target_label: pod
+  - source_labels: [__meta_kubernetes_pod_name]
+    regex: (^[^-]*).*
+    target_label: instance
+    replacement: '$1'
```

Comment:
Is this capturing everything leading up to (and not including) the first dash as `instance`?

Reply:
This is simply a placeholder copy of the `prometheus.yml` configuration file.  Now that it's been included in pgmonitor itself, we will be grabbing it directly from the pgmonitor project (this change will ocurr with the implementation of CH 8765):

https://github.com/CrunchyData/pgmonitor/blob/9697c0d41e2a353f2dda4c353b272e8be570a935/prometheus/crunchy-prometheus.yml.containers

Therefore, we should address any questions/concerns regarding the default Prometheus config with the pgmonitor team.